topics/proteomics/tutorials/metaquantome-function/workflows/main_workflow.ga

```diff
@@ -0,0 +1,623 @@
+            "tool_version": null,
+            "type": "tool",
+            "uuid": "70ec94e4-d11f-4ccb-816d-515d5599cd19",
+            "workflow_outputs": [
+                {
+                    "label": null,
+                    "output_name": "plotdata",
+                    "uuid": "93c4ebae-ea7d-4fb2-85be-6550badd5ba6"
+                },
+                {
+                    "label": null,
+                    "output_name": "wrapped_outfile",
+                    "uuid": "49a776ae-8db5-45b4-9207-2615bcb1ddda"
+                }
+            ]
+        },
+        "10": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_viz/metaquantome_viz/2.0.0-0",
+            "errors": null,
+            "id": 10,
+            "input_connections": {
+                "infile": {
+                    "id": 6,
+                    "output_name": "outfile"
+                },
+                "samps": {
+                    "id": 1,
+                    "output_name": "samples_file"
+                }
+            },
+            "inputs": [
+                {
+                    "description": "runtime parameter for tool metaQuantome: visualize",
+                    "name": "infile"
+                },
+                {
+                    "description": "runtime parameter for tool metaQuantome: visualize",
+                    "name": "samps"
+                }
+            ],
+            "label": "T7-T2-Volcano-Plot",
+            "name": "metaQuantome: visualize",
+            "outputs": [
+                {
+                    "name": "wrapped_outfile",
+                    "type": "html"
+                },
+                {
+                    "name": "plotdata",
+                    "type": "tabular"
+                }
+            ],
+            "position": {
+                "left": 1507.0834655761719,
+                "top": 165.9375057220459
+            },
+            "post_job_actions": {},
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_viz/metaquantome_viz/2.0.0-0",
+            "tool_shed_repository": {
+                "changeset_revision": "02da24eac445",
+                "name": "metaquantome_viz",
+                "owner": "galaxyp",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"height\": \"\", \"infile\": {\"__class__\": \"RuntimeValue\"}, \"mode_args\": {\"mode\": \"f\", \"__current_case__\": 0, \"ontology\": \"go\", \"plot\": {\"plottype\": \"volcano\", \"__current_case__\": 1, \"fc_name\": \"log2fc_T7_over_T2\", \"fc_corr_p\": \"corrected_p_T7_over_T2\", \"textannot\": \"id\", \"flip_fc\": \"false\", \"gosplit\": \"false\"}}, \"samps\": {\"__class__\": \"RuntimeValue\"}, \"width\": \"\", \"__page__\": null, \"__rerun_remap_job_id__\": null}",
```

Comment:
```suggestion
            "tool_state": "{\"height\": \"\", \"infile\": {\"__class__\": \"RuntimeValue\"}, \"mode_args\": {\"mode\": \"f\", \"__current_case__\": 0, \"ontology\": \"go\", \"plot\": {\"plottype\": \"volcano\", \"__current_case__\": 1, \"fc_name\": \"log2fc_T7_over_T2\", \"fc_corr_p\": \"corrected_p_T7_over_T2\", \"textannot\": \"id\", \"flip_fc\": \"false\", \"gosplit\": \"true\"}}, \"samps\": {\"__class__\": \"RuntimeValue\"}, \"width\": \"\", \"__page__\": null, \"__rerun_remap_job_id__\": null}",
```

Reply:
"Make one plot for each of BiologicalProcess, CellularComponent, and MolecularFunction GO term categories." is set to "Yes" according to the tutorial, so please update the parameter in the workflow.